carbon-design-system/carbon
# fix(datepicker): update calendar when range end date is manually changed

Closes #11583 

#### Changelog

**New**

- add new `blur` handler to the end date input, add logic to update/set the calendar

#### Testing / Reviewing

- Go to the range story
- Select a begin and date with the calendar
- Edit the begin date manually
- Blur the input (calendar should be correctly updated to show the new date)
- Edit the end date manually
- Blur the input (calendar should be correctly updated to show the new date)


https://user-images.githubusercontent.com/3360588/213520391-1dad97db-9826-48f2-bedf-8b7c5bdce446.mov
